front/src/common/IntervalsEditor/IntervalsEditor.tsx

```diff
@@ -315,6 +315,7 @@ const IntervalsEditor = (props: IntervalsEditorProps) => {
               setData={setData}
               setSelectedIntervalIndex={setSelected}
               totalLength={totalLength}
+              defaultValue={`${defaultValue}`}
```

Comment:
Change the type and you can remove the cast to string :)
```suggestion
              defaultValue={defaultValue}
```

Reply:
nice ! fixed